crates/store/re_data_source/src/data_source.rs

```diff
@@ -35,6 +34,10 @@ pub enum DataSource {
     /// A file on a Rerun Data Platform server, over `rerun://` gRPC interface.
     #[cfg(feature = "grpc")]
     RerunGrpcUrl { url: String },
+
+    /// A catalog containing metadata information about recordings stored on Rerun Data Platform.
+    #[cfg(feature = "grpc")]
+    RerunGrpcCatalog { url: String },
```

Comment:
A separate source here seems unnecessary, `RerunGrpcUrl` could support both variants.

Reply:
hm, yeah, not sure why I went this way instead of just differentiating in the grpc client. good suggestion!